lib/postgrex/extensions/bit_string.ex

```diff
@@ -6,7 +6,7 @@ defmodule Postgrex.Extensions.BitString do
   def init(opts), do: Keyword.fetch!(opts, :decode_binary)
 
   def encode(_) do
-    quote location: :keep do
+    quote location: :keep, generated: true do
```

Comment:
This is a common pattern: the code explicitly makes a _definitely_ improper list; this is fine for iodata and saves a cons cell.